src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -526,29 +532,39 @@ private static CodeAction TryCreateCodeAction(Document document, AddImportFixDat
                 _ => throw ExceptionUtilities.Unreachable,
             };
 
-        private static ITypeSymbol GetAwaitInfo(SemanticModel semanticModel, ISyntaxFacts syntaxFactsService, SyntaxNode node)
+        private static ITypeSymbol? GetAwaitInfo(SemanticModel semanticModel, ISyntaxFacts syntaxFactsService, SyntaxNode node)
         {
             var awaitExpression = FirstAwaitExpressionAncestor(syntaxFactsService, node);
 
             var innerExpression = syntaxFactsService.GetExpressionOfAwaitExpression(awaitExpression);
 
+            if (innerExpression is null)
```

Comment:
This is a bonus fix. Callers to `GetAwaitInfo` already handle when it returns null.